packages/alfa-style/src/property/combinator/any.ts

```diff
@@ -0,0 +1,44 @@
+import { Parser } from "@siteimprove/alfa-parser";
+import { Err, Result } from "@siteimprove/alfa-result";
+
+export function any<I, T, E, A extends [] = []>(
```

Comment:
I had a hard time figuring out what this function does (parse a list of components where each component is present zero or one time, in an unspecified order).
I feel the name doesn't reflect that, but it might be a standard combinator name (couldn't find ref…) Plus, I don't really have anything else to suggest…
(maybe just a nice docstring…)

Reply:
I'll add a `@see` annotation linking to https://drafts.csswg.org/css-values/#comb-any. The behaviour of this combinator is indeed a little odd, hence my comment in https://github.com/Siteimprove/alfa/pull/301#discussion_r468452732.